lib/common/mocks/Certification.go

```diff
@@ -0,0 +1,55 @@
+package mocks
+
+import "github.com/opencontrol/compliance-masonry/lib/common"
+import "github.com/stretchr/testify/mock"
+
+// Certification is an autogenerated mock type for the Certification type
```

Comment:
yeah via [mockery](https://github.com/vektra/mockery). Probably should put something in the development documentation about how to generate the mocks.